mm2src/coins/lp_coins.rs

```diff
@@ -1103,17 +1125,17 @@ pub trait GetWithdrawSenderAddress {
     ) -> MmResult<WithdrawSenderAddress<Self::Address, Self::Pubkey>, WithdrawError>;
 }
 
-#[derive(Clone, Deserialize)]
+#[derive(Clone, Deserialize, Serialize)]
 #[serde(untagged)]
 pub enum WithdrawFrom {
-    // AccountId { account_id: u32 },
     AddressId(HDAccountAddressId),
     /// Don't use `Bip44DerivationPath` or `RpcDerivationPath` because if there is an error in the path,
     /// `serde::Deserialize` returns "data did not match any variant of untagged enum WithdrawFrom".
     /// It's better to show the user an informative error.
     DerivationPath {
         derivation_path: String,
     },
+    HDWalletAddress(StandardHDCoinAddress),
```

Comment:
Just for thinking about:

Is it reasonable to rename it in more deliberate way, something like `Bip44Address(Bip44AddressSuffix)` could be better

Reply:
https://github.com/KomodoPlatform/komodo-defi-framework/pull/1933#discussion_r1287859252

Reply:
As I can see bip44 is only one of used derivation schemas, there are others too. So referring to a particular standard may be good idea. However as I can tell bitcoin uses just one bip32 path format for all schemas what seems feasible.
Also name StandardHDCoinAddress seems not to be quite accurate as the path refers primarily to the key than address